spec/integration/aws_route_table_spec.rb

```diff
@@ -84,6 +84,78 @@
         ).and be_idempotent
       end
 
+      it "updates an existing routing table" do
```

Comment:
To make this a little more clear, and because we're already testing create logic in another test, this should be rewritten as

```ruby
context "with an existing routing table" do
  aws_route_table 'test_route_table' do
    vpc 'test_vpc'
    routes '0.0.0.0/0' => :internet_gateway
  end

  it "updates an existing routing table" do
    expect_recipe {
      aws_route_table 'test_route_table' do
        vpc 'test_vpc'
        routes '0.0.0.0/0' => :internet_gateway,
               '10.1.0.0/24' => :internet_gateway
      end
    }.to update_an_aws_route_table('test_route_table',
      routes: [
        { destination_cidr_block: '10.1.0.0/24', gateway_id: test_vpc.aws_object.internet_gateway.id, state: "active" },
        { destination_cidr_block: '10.0.0.0/24', gateway_id: 'local', state: "active" },
        { destination_cidr_block: '0.0.0.0/0', gateway_id: test_vpc.aws_object.internet_gateway.id, state: "active" },
      ]
    ).and be_idempotent
  end
end
```